src/pages/Product/ProductList/ProductFilterTab/FilterSort/FilterSort.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import '../FilterSort/FilterSort.scss';
+
+function FilterSort({ sortProducts, sortReview, sortReputation }) {
+  return (
+    <ul>
+      <form>
+        <li>
+          <button type="button" onClick={() => sortProducts()}>
```

Comment:
```suggestion
          <button type="button" onClick={sortProducts}>
```
* 이벤트 함수에 콜백으로 넘겨줄 인자가 없다면, 함수 호출문이 아닌, 함수 자체를 넣어주세요